### PR TITLE
[stable/sematext-agent] Updates to auto discovery configuration file in sematext agent

### PR DESCRIPTION
**Pull request by sivasamyk**
The PR updates the variable names in the autodiscovery configuration file of sematext agent chart, based on recent changes in App Agent repo

@komljen 

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/sematext-agent/files/autodisco.yml [hunk 1]
```diff
@@ -75,7 +75,7 @@ couchbase:
     - 8094
     - 11210
   labels:
-    COUCHBASE_HOSTPORT: "{{ .IP }}:8093"
+    COUCHBASE_HOST_PORT: "{{ .IP }}:8093"
     COUCHBASE_QUERY_HOSTPORT: "{{ .IP }}:8093"
```
Comment (hakman) [inline stable/sematext-agent/files/autodisco.yml:79]: Change to `HOST_PORT` here also?

Comment (hakman) [reply stable/sematext-agent/files/autodisco.yml:79]: Also in SPM_MONITOR_ES_NODE_HOSTPORT?

**Comment by sivasamyk** [comment]
Closing this PR since https://github.com/helm/charts/pull/15310/ has latest changes

**Pull request closed without merging**